videointelligence/tests/system.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2017, Google LLC All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""System tests for VideoIntelligence API."""
+
+import grpc
+import io
+import json
+import os
+import requests
+import time
+import unittest
+
+import google.api_core.exceptions
+from google.cloud import exceptions
+from google.cloud import videointelligence
+from google.cloud.videointelligence_v1 import enums
+
+PROJECT_NUMBER = os.environ.get("PROJECT_NUMBER")
+PROJECT_OUTSIDE = os.environ.get("GOOGLE_CLOUD_TESTS_VPCSC_OUTSIDE_PERIMETER_PROJECT")
+BUCKET_OUTSIDE = os.environ.get("GOOGLE_CLOUD_TESTS_VPCSC_OUTSIDE_PERIMETER_BUCKET")
+OUTSIDE_PROJECT_API_KEY = os.environ.get(
```

Comment:
These variables aren't set in our environment. If you'd like these tests to run as part of our CI, please set them in the noxfile (see the VPCSC PR for Assset)

https://github.com/googleapis/google-cloud-python/pull/8613/files

Reply:
Done.